tslint_eslint_rules.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@buzinas I noticed that there is a file called `eslint_tslint.json` with a list of the rules in the project. Where those meant to be the preferences for the source code in this project? I tried them but it gave a lot of errors so I'll be adding some rules here and there as I see them fit. For now I started with the obvious ones. Is it safe to delete `eslint_tslint.json`?